posthog/year_in_posthog/hibernating.html

```diff
@@ -0,0 +1,145 @@
+            font-style: normal;
+            font-weight: 700;
+            font-display: swap;
+            src: url('{% static 'Inter.woff2' %}') format('woff2'), url('{% static 'Inter.woff' %}') format('woff');
+            unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC,
+            U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+        }
+
+        @font-face {
+            font-family: 'Inter';
+            font-style: normal;
+            font-weight: 500;
+            font-display: swap;
+            src: url('{% static 'Inter.woff2' %}') format('woff2'), url('{% static 'Inter.woff' %}') format('woff');
+            unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC,
+            U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+        }
+
+        /* Inter; normal (400); latin */
+        @font-face {
+            font-family: 'Inter';
+            font-style: normal;
+            font-weight: 400;
+            font-display: swap;
+            src: url('{% static 'Inter.woff2' %}') format('woff2'), url('{% static 'Inter.woff' %}') format('woff');
+            unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC,
+            U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+        }
+
+        html {
+            font-family: "Inter", -apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica, sans-serif;
+            height: 100%;
+            width: 100%;
+        }
+
+        body {
+            background-color: #2D2D2D;
+            color: #fff;
+            display: flex;
+            flex-direction: column;
+            max-width: 1280px;
+            align-items: center;
+            justify-content: center;
+            margin: auto;
+            padding: 5rem 1rem;
+        }
+
+        body:before {
+            content: ' ';
+            display: block;
+            position: absolute;
+            left: 0;
+            top: 0;
+            width: 100%;
+            height: 100%;
+            opacity: 0.1;
+            background-image: url("{%  static "background.png" %}");
+            background-repeat: no-repeat;
+            background-position: 50% 0;
+            background-size: cover;
+        }
+
+        h1 {
+            margin: 0;
+            text-align: center;
+        }
+
+        @media only screen and (min-width: 650px) {
+            h1 {
+                margin-bottom: -4rem;
+            }
+        }
+
+        img {
+            width: 100%;
+        }
+
+        @media only screen and (min-width: 650px) {
+            img {
+                max-width: 800px;
+            }
+        }
+
+    </style>
+</head>
+<body>
+
+<h1>Year in PostHog is hibernating.</h1>
+<img src="{% static 'sleeping-hog.png' %}" alt="Year in PostHog is hibernating."/>
```

Comment:
Ah noticed one thing, we should be specifying width and height for this element:
1. So that this isn't blurry on Retina displays (so width 400px)
1. To minimize any cumulative layout shift, as a best practice